### ISSUE TITLE
lint for generic parameters in comments

### ISSUE THREAD

**Issue by hterkelsen** [dart-lang/linter#624]
GOOD:
```dart
T identity<T>(T x) => x;
```

BAD:
```dart
dynamic/*=T*/ identity/*<T>*/(dynamic/*=T*/ x) => x;
```

**Comment by alexeieleusis** [comment]
With a quickfix!

**Comment by srawlins** [comment]
This syntax is going away very soon. Just FYI. https://github.com/dart-lang/sdk/issues/28796

**Comment by bwilkerson** [comment]
We already have a hint for this.
